t2gether/urls.py

```diff
@@ -13,7 +13,8 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
+    path('user/', include('user.urls')),
```

Comment:
```suggestion
    path('user', include('user.urls')),
```
main url에서는 앱 이름 양쪽으로 슬래시 붙이지 않고, 앱 url에서 앞에 슬래시 붙이는 것이 위코드 코딩 컨벤션입니다 :)